src/main/java/io/arivera/oss/embedded/rabbitmq/download/BasicDownloader.java

```diff
@@ -7,7 +7,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.File;
 import java.io.IOException;
+import java.io.InputStream;
+import java.net.Proxy;
+import java.net.URL;
+import java.net.URLConnection;
```

Comment:
(I've updated the master branch to add the CheckStyle rule that enforces this. I'm not sure why this wasn't there before)